src/js/services/storageService.js

```diff
@@ -219,6 +219,14 @@ angular.module('copayApp.services')
       storage.set('homeTip', val, cb);
     };
 
+    root.getBuyNavTipAccepted = function(cb) {
+      storage.get('buyNavTip', cb);
+    };
+
+    root.setBuyNavTipAccepted = function(val, cb) {
+      storage.set('buyNavTip', val, cb);
+    };
+
```

Comment:
Was pleasantly surprised to see this feature added 🙂

Reply:
Was annoying me. So just did it at the same time. Should have actually done in a different PR. But... oh well.